library/ecp.c

```diff
@@ -2061,6 +2061,57 @@ int mbedtls_ecp_check_pub_priv( const mbedtls_ecp_keypair *pub, const mbedtls_ec
     return( ret );
 }
 
+#if defined(MBEDTLS_ASN1_WRITE_C) && defined(MBEDTLS_OID_C)
+#include "mbedtls/asn1write.h"
+#include "mbedtls/oid.h"
+int mbedtls_ecp_ansi_write_group( const mbedtls_ecp_group *grp,
+                                  unsigned char *p,
+                                  size_t size )
+{
+    const char *oid;
+    unsigned char *q;
+    size_t oid_length;
+    int ret;
+    ret = mbedtls_oid_get_oid_by_ec_grp( grp->id, &oid, &oid_length );
```

Comment:
Style: can we have some blank lines please? At least between variable declarations and code. (Same below.)

Reply:
Done